pandas/core/strings.py

```diff
@@ -1314,21 +1314,65 @@ def str_index(arr, sub, start=0, end=None, side='left'):
 
 def str_pad(arr, width, side='left', fillchar=' '):
     """
-    Pad strings in the Series/Index with an additional character to
-    specified side.
+    Pad strings in the Series/Index up to width.
 
```

Comment:
I think in this case we could have a extended summary, providing a bit more information on what pad means, and when it can be useful.